viewing_party/party.py

```diff
@@ -1,23 +1,168 @@
+# -----------------------------------------
 # ------------- WAVE 1 --------------------
+# -----------------------------------------
+from csv import list_dialects
+
+
+TITLE_KEY = "title"
+GENRE_KEY = "genre"
+RATING_KEY = "rating"
+
 
 def create_movie(title, genre, rating):
-    pass
+    new_movie = {
+        TITLE_KEY: title,
+        GENRE_KEY: genre,
+        RATING_KEY: rating}
+
+    if not new_movie[TITLE_KEY]:
+        return None
+    if not new_movie[GENRE_KEY]:
+        return None
+    if not new_movie[RATING_KEY]:
+        return None
+    return new_movie
+
+
+def add_to_watched(user_data, movie):
+    user_data = {"watched": [movie]}

```

Comment:
I was reading through the code because I have some time to review and I was thinking this doesn't look right at all for the function name, 'add'. This confirmed it. Made the change